### PR TITLE
Replaces OOC in IC with schizophrenia

### PR DESCRIPTION
**Pull request by ChungusGamer666**
## About The Pull Request

Adds schizohelp, a basic mentoring system players can opt into.
image
image

## Why It's Good For The Game

I can finally frag people for OOC in IC.
image

### CODE REVIEW

File: code/modules/mob/living/say.dm [hunk 1]
```diff
@@ -103,6 +103,15 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
...
 		SSblackbox.record_feedback("tally", "ic_blocked_words", 1, lowertext(config.ic_filter_regex.match))
 		return
+	
+	var/static/regex/ooc_regex = regex(@"\((.*?)\)")
+	if(findtext(message, ooc_regex))
+		emote("me", 1, "mumbles incoherently.")
+		to_chat(src, "<span class='warning'>That was stupid of me. I should meditate on my actions.</span>")
+		message_admins("[key_name_admin(src)] tried to say an OOC message IC! Laugh at this loser!")
+		add_stress(/datum/stressevent/ooc_ic)
+		adjust_playerquality(-1, src.ckey)
```
Comment (Real-MAGNUM) [inline code/modules/mob/living/say.dm:113]:
```suggestion
```

**Comment by Real-MAGNUM** [comment]
Removed -1 pq for ((

**Comment by BlueMemesauce** [comment]
We already said that adding a filter for (
Is just gonna make people use another symbol or maybe just say It without any symbol
Any plans to address this?

**Comment by BlueMemesauce** [comment]
And how do you plan to deal with people saying halp what does revolution mean, revealing that their is a revolution?

**Comment by BlueMemesauce** [comment]
And this is just gonna make the pq system pointless since you can rack up so much pq so quickly using this

**Comment by Real-MAGNUM** [comment]
> We already said that adding a filter for ( Is just gonna make people use another symbol or maybe just say It without any symbol Any plans to address this?

It's mentioned in the announcements. Rules will reflect it. New things that catch on will be added to the blacklist

File: code/modules/mob/living/emote.dm [hunk 2]
```diff
@@ -101,7 +102,27 @@
...
+	restraint_check = FALSE
+	emote_type = EMOTE_VISIBLE
+
+/mob/living/carbon/human/verb/emote_meditate()
+	set name = "Meditate"
+	set category = "Emotes"
+
+	emote("meditate", intentional = TRUE)
+
+/datum/emote/living/pray/run_emote(mob/user, params, type_override, intentional)
```
Comment (BlueMemesauce) [inline code/modules/mob/living/emote.dm:121]: also this

File: code/modules/mob/living/emote.dm [hunk 3]
```diff
@@ -101,7 +102,27 @@
 			return TRUE
 		else
 			return TRUE
-	else to_chat(L, "<span class='danger'>My prayer was kinda short...</span>")
+	else 
+		to_chat(L, "<span class='danger'>My prayer was kinda short...</span>")
+
+/datum/emote/living/pray
```
[the remark this answers is not in the document]
Comment (Real-MAGNUM) [reply code/modules/mob/living/emote.dm:108]: It did.

File: code/modules/mob/living/say.dm [hunk 4]
```diff
@@ -103,6 +103,14 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 		to_chat(src, "<span class='warning'>That message contained a word prohibited in IC chat! Consider reviewing the server rules.\n<span replaceRegex='show_filtered_ic_chat'>\"[message]\"</span></span>")
 		SSblackbox.record_feedback("tally", "ic_blocked_words", 1, lowertext(config.ic_filter_regex.match))
 		return
+	
+	var/static/regex/ooc_regex = regex(@"\((.*?)\)")
```
Comment (efzapa) [inline code/modules/mob/living/say.dm:107]: replace regex with `^\(.*|.*\)$|^\(.*\)$`

**Review by efzapa — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/emote.dm (hunk 3 above)
[the remark this answers is not in the document]
Comment (ChungusGamer666) [reply code/modules/mob/living/emote.dm:108]: i forgor

**Pull request merged**